includes/namespace.php

```diff
@@ -80,6 +80,35 @@ function deactivate( $network_wide ) {
 register_activation_hook( WEBSTORIES_PLUGIN_FILE, '\Google\Web_Stories\activate' );
 register_deactivation_hook( WEBSTORIES_PLUGIN_FILE, '\Google\Web_Stories\deactivate' );
 
+if (
+	! class_exists( '\Google\Web_Stories\Plugin' ) ||
+	! file_exists( WEBSTORIES_PLUGIN_DIR_PATH . '/assets/js/edit-story.js' )
+) {
+	add_action(
+		'admin_notices',
+		static function () {
```

Comment:
I thought we were removing static functions in this code base. They are horrible for readiblity.

Reply:
You mean anonymous functions / closures? In this instance I am not sure a proper function is needed, as we don't test this file and removing this action shouldn't really be possible either.

Reply:
I know we are not testing this, but I still don't like them for style either, they are ugly are hard to read and impossible to unhook. Just don't like to see them. But this is a nit pic.